tests/func/test_diff.py

```diff
@@ -1,267 +1,86 @@
-import os
+import pytest
 
-import dvc.repo.diff as diff
-from dvc.main import main
-from dvc.scm.base import FileNotInCommitError
-from tests.basic_env import TestDvcGit
 
+def test_no_scm(tmp_dir, dvc):
+    tmp_dir.dvc_gen("file", "text")
 
-class TestDiff(TestDvcGit):
-    def setUp(self):
-        super().setUp()
+    pytest.skip("TODO: define behavior, should it fail?")
```

Comment:
So this is a WIP PR, right?

Reply:
It is, but it would be great to know how should `dvc diff` should handle `NoSCM` cases.